api/cfnetworking/cfnetv1/policy.go

```diff
@@ -52,3 +53,52 @@ func (client Client) CreatePolicies(policies []Policy) error {
 
 	return client.connection.Make(request, &cfnetworking.Response{})
 }
+
+// ListPolicies will list the policies
+func (client Client) ListPolicies(appGUIDs ...string) ([]Policy, error) {
+	var request *cfnetworking.Request
+	var err error
+	if len(appGUIDs) == 0 {
+		request, err = client.newHTTPRequest(requestOptions{
+			RequestName: internal.ListPolicies,
+		})
+	} else {
+		request, err = client.newHTTPRequest(requestOptions{
+			RequestName: internal.ListPolicies,
+			Query: map[string][]string{
+				"id": {strings.Join(appGUIDs, ",")},
```

Comment:
if your API accepts and empty `id` then you won't need to have an if/else block. But it depends on how your API handles it...

Reply:
It looks like we will need to keep the if/else clause, the API will always return an empty policy list if we pass an empty string as a query param